easybuild/easyblocks/generic/buildenv.py

```diff
@@ -28,14 +28,35 @@
 
 @author: Alan O'Cais (Juelich Supercomputing Centre)
 """
+import os
+
 from easybuild.easyblocks.generic.bundle import Bundle
+from easybuild.tools.config import build_option, update_build_option
+from easybuild.tools.toolchain.toolchain import RPATH_WRAPPERS_SUBDIR
 
 
 class BuildEnv(Bundle):
     """
     Build environment of toolchain: only generate module file
     """
 
+    def prepare_step(self, *args, **kwargs):
+        """
+        Custom prepare step for buildenv: export rpath wrappers if they are being used
+        """
+        # We export the rpath wrappers under special conditions
+        # (the wrappers are needed if LD_LIBRARY_PATH is being filtered)
+        filtered_env_vars = build_option('filter_env_vars') or []
+        if build_option('rpath') and 'LD_LIBRARY_PATH' in filtered_env_vars and 'LIBRARY_PATH' not in filtered_env_vars:
```

Comment:
Seems strict. Even if you want to easy into getting rid of LD_LIBRARY_PATH, you don't get to have this wrapper(?)

I suppose the last condition is because LIBRARY_PATH is required by rpath wrapper?

Reply:
This is the scenario where you would actually need it, if you have `LD_LIBRARY_PATH` the wrappers aren't really necessary in the first place.

Having said that, if you ask for them it should probably just provide them. Maybe a better approach is to warn that you actually need them in that specific scenario.

Reply:
I'll add an option to which can manually trigger this.

Reply:
I would simply always install the RPATH wrapper scripts (if they exist), regardless of how `filter-env-vars` is configured.

`$LIBRARY_PATH` may not be set by the modules generated by EasyBuild, but it can still be set manually. And the RPATH wrappers will do useful stuff even if `$LIBRARY_PATH` is not defined.